src/main/scala/no/ndla/listingapi/controller/ListingController.scala

```diff
@@ -99,6 +109,16 @@ trait ListingController {
       searchService.matchingQuery(filter, language, page.toInt, pageSize.toInt, sort)
     }
 
+
+    get("/theme/:theme", operation(getThemeDoc)) {
+      val theme = params("theme")
+      val language = paramOrDefault("language", DefaultLanguage)
+      theme match {
+        case theme if Theme.allowedThemes.contains(theme) => readService.getTheme(theme, language)
+        case _ => NotFound(body = Error(Error.NOT_FOUND, s"No theme with name '$theme' is configured."))
```

Comment:
Hadde det vært lurt med en case som trigges dersom `theme` ikke er i `Theme.allowedThemes` slik at det kan returneres en validation error i stedet for en 404?